.travis.yml

```diff
@@ -99,7 +99,15 @@ script:
       echo "Skipping script build step in Coverity branch/job";
     else
       travis_retry travis_wait 50 \
-          mvn clean verify --projects janusgraph-${MODULE} ${ARGS};
+          mvn clean verify --projects janusgraph-${MODULE} -Pcoverage ${ARGS};
+    fi
+
+after_success:
+  # Upload test coverage reports to Codecov
+  - if [ "${COVERITY_SCAN_BRANCH}" == 1 ] || [ -v COVERITY_ONLY ]; then
```

Comment:
@sjudeng — I think I must have missed this line in the original review of this PR (and this file hasn't changed since then), but what's the `-v` test do? I can't find it in the [`bash(1)` man page](http://man7.org/linux/man-pages/man1/bash.1.html#CONDITIONAL_EXPRESSIONS).

Or did you perhaps mean to use `[ -z "${COVERITY_ONLY:-}" ]` as is done elsewhere in this file?

Of course, it's typically used in the negated form: `! [ -z "${COVERITY_ONLY:-}" ]`; the single-character equivalent to that is `[ -n "${COVERITY_ONLY:-}" ]` if that's what you're looking for.

Reply:
@mbrukman I might have missed this when you provided original feedback. I agree, the check here should be something like the below. 

```
  - if [ "${TRAVIS_BRANCH}" == "${COVERITY_BRANCH_NAME}" ] || ! [ -z "${COVERITY_ONLY:-}" ]; then
      echo "Skipping test coverage report upload in Coverity branch/job";
    else
      bash <(curl -s https://codecov.io/bash);
    fi
```

I can get a fix in but it might be a bit or else feel free to beat me to it. Currently reports are uploading to codecov properly. It's just attempting (and failing) to upload non-existent reports on regular module jobs on the coverity_scan branch (see https://travis-ci.org/JanusGraph/janusgraph/jobs/279705005#L488).

Reply:
Sounds good; PR #761 sent with the fix.